### DISCUSSION TITLE
Opting in to receiving next-gen node_id in webhooks?

### DISCUSSION THREAD

**Discussion by lewisl9029** [community/community#20068] [Other Feature Feedback, Questions, & Ideas]
Hi there! I've been building a Github App that uses [next-gen `node_id`s](https://docs.github.com/en/graphql/guides/migrating-graphql-global-node-ids#migrating-to-the-new-global-ids) across the board when requesting data from the Github API, using the `X-Github-Next-Global-ID` header.

This has been working great, until I noticed that webhooks are still sending me last-gen node_ids for some nodes, so I have to translate that into a next-gen ID by making a separate request and adding another roundtrip. 

Would be awesome to be able to opt in to receiving next-gen node_ids in webhooks so we don't get penalized in terms of latency for making this migration. Thanks! :)

**Comment by leipert** [comment]
Same request: https://github.com/orgs/community/discussions/147848

**Comment by antonkomarev** [comment]
Any updates on this?
